extensions/adobe/experience/target/classification.schema.json

```diff
@@ -0,0 +1,97 @@
+                },
+                "xdm:experienceName": {
+                    "title": "Experience Name",
+                    "type": "string",
+                    "description": "The name of the Target experience."
+                },
+                "xdm:activity>experience": {
+                    "title": "Activity > Experience",
+                    "type": "string",
+                    "description": "Target Activity name concatenated with experience name."
+                },
+                "xdm:thirdId": {
+                    "title": "Third Id",
+                    "type": "string",
+                    "description": "The third id of the classification record."
+                },
+                "xdm:activityId": {
+                    "title": "Activity Id",
+                    "type": "string",
+                    "description": "The identity of Target activity."
+                },
+                "xdm:experienceId": {
+                    "title": "Experience Id",
+                    "type": "string",
+                    "description": "The identity of Target experience."
+                },
+                "xdm:analyticsReporting": {
+                    "title": "Analytics Reporting",
+                    "type": "string",
+                    "description": "The reporting type associated with the Target activity."
+                },
+                "xdm:activeDates": {
+                    "title": "Active Dates",
+                    "type": "string",
+                    "description": "The date of Target activity activation."
+                },
+                "xdm:controlExperience": {
+                    "title": "Control Experience",
+                    "type": "string",
+                    "description": "The experience set as control experience in the Target activity."
+                },
+                "xdm:algorithmId": {
+                    "title": "Algorithm Id",
+                    "type": "string",
+                    "description": "The identity of the algorithm used in the Target activity."
+                },
+                "xdm:controlvsTargeted": {
+                    "title": "Control vs Targeted",
+                    "type": "string",
+                    "description": "The traffic allocation type for the Target activity."
+                },
+                "xdm:trafficTypeId": {
+                    "title": "Traffic Type Id",
+                    "type": "string",
+                    "description": "The identity of the traffic type associated with Target activity."
+                }
+            },
+            "required": [
+                "xdm:Key",
+                "xdm:ActivityName",
+                "xdm:ActivityId",
+                "xdm:ExperienceName",
+                "xdm:ExperienceId"
+            ]
+        }
+    },
+    "allOf": [
+        {
+            "$ref": "#/definitions/classification"
+        }
+    ],
+    "meta:status": "stable"
```

Comment:
You should mark this as experimental for this first PR. https://github.com/adobe/xdm/blob/master/CONTRIBUTING.md#specify-the-metastatus